cmd/kubeadm/app/util/config/initconfiguration.go

```diff
@@ -110,6 +110,11 @@ func SetNodeRegistrationDynamicDefaults(cfg *kubeadmapi.NodeRegistrationOptions,
 	}
 
 	if cfg.CRISocket == "" {
+		if skipCRIDetect {
+			klog.V(4).Infof("skip CRI socket detection, fill with placeholder %s", kubeadmconstants.UnknownCRISocket)
```

Comment:
pls have some info about the phase/command/func name in the log, this will help us to quickly identify where we are since the same log is print inside `SetResetDynamicDefaults`.

Reply:
`SetResetDynamicDefaults` is only called by `LoadOrDefaultResetConfiguration`.
So we can tell where we are from the top context.
For example:
```sh
I1007 07:24:17.900137    4332 initconfiguration.go:260] loading configuration from "./kubeadm.yaml"
I1007 07:24:17.900851    4332 initconfiguration.go:114] skip CRI socket detection, fill with placeholder unix:///var/run/unknown.sock
I1007 07:24:17.902233    4332 interface.go:432] Looking for default routes with IPv4 addresses
I1007 07:24:17.902269    4332 interface.go:437] Default route transits interface "eth0"
I1007 07:24:17.902387    4332 interface.go:209] Interface eth0 is up
I1007 07:24:17.902482    4332 interface.go:257] Interface "eth0" has 3 addresses :[172.18.0.2/16 fc00:f853:ccd:e793::2/64 fe80::42:acff:fe12:2/64].
I1007 07:24:17.902542    4332 interface.go:224] Checking addr  172.18.0.2/16.
I1007 07:24:17.902557    4332 interface.go:231] IP found 172.18.0.2
I1007 07:24:17.902601    4332 interface.go:263] Found valid IPv4 address 172.18.0.2 for interface "eth0".
I1007 07:24:17.902610    4332 interface.go:443] Found active IP 172.18.0.2 
...
```